### ISSUE TITLE
XML validation exception: missing schema in \src\it\settings.xml

### ISSUE THREAD

**Issue by dpisarenko** [jcabi/jcabi-ssl-maven-plugin#7]
When I run `mvn qulice:check -Pqulice`, I get the error message `XML validation exception: missing schema in \src\it\settings.xml`.

Full output:

```
$ mvn qulice:check -Pqulice
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building jcabi-ssl-maven-plugin 1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- qulice-maven-plugin:0.8.2:check (default-cli) @ jcabi-ssl-maven-plugi
n ---
[INFO] LICENSE found: file:d:/dev/teamed.io/jcabi-ssl-maven-plugin/LICENSE.txt
log4j:WARN No appenders could be found for logger (org.apache.commons.beanutils.
converters.BooleanConverter).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more in
fo.
[INFO] No Checkstyle violations found in 21 files
[INFO] No PMD violations found in 10 files
[INFO] \src\changes\changes.xml: to be validated
[ERROR] JSR-303 validator failed to initialize: Unable to create a Configuration
, because no Bean Validation provider could be found. Add a provider like Hibern
ate Validator (RI) to your classpath. (see http://www.jcabi.com/jcabi-aspects/js
r-303.html)
[INFO] jcabi-aspects 0.15/1409f87 started new daemon thread jcabi-loggable for w
atching of @Loggable annotated methods
[INFO] \src\it\end-to-end\pom.xml: to be validated
[INFO] \src\it\settings.xml: to be validated
[INFO] Read our quality policy: http://www.qulice.com/quality.html
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 20.055 s
[INFO] Finished at: 2014-12-28T03:21:58+04:00
[INFO] Final Memory: 15M/49M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.qulice:qulice-maven-plugin:0.8.2:check (defau
lt-cli) on project jcabi-ssl-maven-plugin: Failure: XML validation exception: mi
ssing schema in \src\it\settings.xml -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e swit
ch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please rea
d the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureExc
eption
```

Please fix this problem by either adding a schema to `src/it/settings.xml` or excluding it from XML validation.

**Comment by dmarkov** [comment]
@dpisarenko thanks a lot for reporting, **15 mins** added to your acc, pmt ID `49703690`

**Comment by palyuga** [comment]
@dpisarenko looks like you've already fix the problem in PR #5
@dmarkov we can close this issue

**Comment by yegor256** [comment]
@palyuga please ask @dpisarenko to close, he is the reporter

**Comment by palyuga** [comment]
@dpisarenko close this, please

**Comment by dmarkov** [comment]
@palyuga just a reminder that you're working with this task for 19 days already... -30 added to your rating, at the moment it is: +150

**Comment by palyuga** [comment]
@dpisarenko ping. Let's close this as you have fixed the issue in PR #5

**Comment by dmarkov** [comment]
@palyuga Much appreciated! **30 mins** was added to your account, payment ID `50573642`. +30 added to your rating, current score is: +150
